docs/content/config/environment.md

```diff
@@ -62,6 +62,15 @@ Amavis content filter (used for ClamAV & SpamAssassin)
 - 1/2      => Show default informational output
 - 3/4/5    => log debug information (very verbose)
 
+##### ENABLE_DNSBL
+
+This enables the [zen.spamhaus.org](https://www.spamhaus.org/zen/) DNS block list.
+
+Note: Emails will be rejected, if they don't pass the spamhaus check!
+
+- **0** => DNS block list is disabled
+- 1     => DNS block list is enabled
```

Comment:
We _could_ make `1` the default, and with an upcoming version, that is not a patch version, change the default. This would provide a "grace period" until, let's say, `v10.5` is released. We would therefore circumvent a real change with a patch version. What do you think?

Reply:
I've not considered this for the patch release, see the milestone label 😉 Patch releases should only contain patches and not introduce new stuff.
Another option would be, to skip the patch release and directly release `10.5`.

Reply:
That is one way, and certainly a valid one. Are the changes / patches enough to justify a new release though?

Reply:
Sorry, I was quiet busy the last days. It depends on how urgent the ldap patch is. If urgent, then a patch release should be made asap. If it's not that critical, we can wait until some more changes are there for a minor update.